wordpress-mobile/gutenberg-mobile
# Display radial gradient in block which supporting gradients

Fixes missing possibility to display radial-gradients in Buttons or Cover blocks.

Ref to gutenberg: https://github.com/WordPress/gutenberg/pull/22384

To test:

1. Open draft on web
2. Add `Buttons` or `Cover`
3. Set radial gradient background color
4. Save draft
5. Open post on mobile
6. Expect to see radial gradient

PR submission checklist:
